src/components/icons/camera/Camera.tsx

```diff
@@ -0,0 +1,16 @@
+import { DEFAULT_ICON_SIZE } from '../.././../models';
+import React from 'react';
+import { Icon, SVGProps } from '../../icon';
+
+export const Camera = ({ title = 'Camera', size = DEFAULT_ICON_SIZE, color }: SVGProps) => (
+  <Icon title={title}>
+    <svg width={size} height={size} viewBox="0 0 14 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```

Comment:
I think we need to rename attributes a bit
```suggestion
        fillRule="evenodd"
        clipRule="evenodd"
```

Reply:
Good one! I've changed it.